src/video_emotion/emotionTagger/face_emotion_extraction.py

```diff
@@ -2,22 +2,61 @@
 import numpy
 from keras.models import load_model
 from keras import backend as K
+from typing import Dict
 
 from utilities.filehandler.handle_path import get_absolute_path
 
 
-def get_labels():
+def get_labels() -> Dict[int, str]:
+
+    """Get the labels for each argument in the emotion array
+
+        Returns
+        -------
+        string
```

Comment:
This should be indented the same amount as the """ like so:
```
    """Get the labels for each argument in the emotion array
    Returns
    -------
    string
```

Also, the return type doesn't match the type hinted in the def